src/api/app/assets/javascripts/webui/application/comment.js

```diff
@@ -28,6 +28,7 @@ function reloadCommentBindings() {
   $('.togglable_comment').click(function () {
       var toggleid = $(this).data("toggle");
       $("#" + toggleid).toggle();
+      $("#" + toggleid).toggleClass('d-none');
```

Comment:
This shouldn't be needed since the line above already deals with the visiblity.

Reply:
@dmarcoux, is needed because this js file is under webui and not webui2.
Because of `d-none` class add `display: none !important`, the line above doesn't  work, so we need to toogle the class to keep the compatibility with `webui`.

Reply:
Alright! Thank you for the explanation.